src/libraries/System.Security.Cryptography.X509Certificates/tests/DynamicChainTests.cs

```diff
@@ -832,6 +833,56 @@ public static void PolicyConstraints_Mapped()
             }
         }
 
+        [Fact]
+        [OuterLoop("Modifies a shared trust store.")]
```

Comment:
I think it makes sense for this test to be outer loop, but maybe I am being too cautious.

The test attempts to be clean up before and after itself. The intermediate certificates get a marker extension for the test, and cleaned up before and after test execution.